{
  "Header": {
    "Original": "CVE-2020-14387 (A flaw was found in rsync in versions since 3.2.0pre1. Rsync improperl ...)",
    "Line": 190934,
    "ID": "CVE-2020-14387",
    "Description": "(A flaw was found in rsync in versions since 3.2.0pre1. Rsync improperl ...)"
  },
  "Annotations": [
    {
      "Original": "- rsync 3.2.3-3 (bug #969530)",
      "Line": 190935,
      "Type": "package",
      "Package": "rsync",
      "Kind": "fixed",
      "Version": "3.2.3-3",
      "BugNo": 969530
    },
    {
      "Original": "[buster] - rsync \u003cnot-affected\u003e (Vulnerable code introduced later)",
      "Line": 190936,
      "Type": "package",
      "Release": "buster",
      "Package": "rsync",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced later"
    },
    {
      "Original": "[stretch] - rsync \u003cnot-affected\u003e (Vulnerable code introduced later)",
      "Line": 190937,
      "Type": "package",
      "Release": "stretch",
      "Package": "rsync",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced later"
    },
    {
      "Original": "NOTE: Introduced by: https://git.samba.org/?p=rsync.git;a=commitdiff;h=2a87d78f693f10fe5ad13af0bb9311bd3714077d (v3.2.0pre1)",
      "Line": 190938,
      "Type": "NOTE",
      "Description": "Introduced by: https://git.samba.org/?p=rsync.git;a=commitdiff;h=2a87d78f693f10fe5ad13af0bb9311bd3714077d (v3.2.0pre1)"
    },
    {
      "Original": "NOTE: Fixed by: https://git.samba.org/?p=rsync.git;a=commitdiff;h=c3f7414c450faaf6a8281cc4a4403529aeb7d859",
      "Line": 190939,
      "Type": "NOTE",
      "Description": "Fixed by: https://git.samba.org/?p=rsync.git;a=commitdiff;h=c3f7414c450faaf6a8281cc4a4403529aeb7d859"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1875549",
      "Line": 190940,
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=1875549"
    }
  ]
}